src/pages/index.js

```diff
@@ -53,67 +54,206 @@ newCardPopup.setEventListeners();
 // New Popup Form to edit profile
 const editProfilePopup = new PopupWithForm(
   "#profile-edit-modal",
-  (formData) => {
-    const title = formData.title;
-    const description = formData.description;
-    userInfo.setUserInfo(title, description);
-    handleProfileEditSubmit(formData);
-  }
+  handleProfileEditSubmit
 );
 editProfilePopup.setEventListeners();
 
 // New Popup with Image
 const popupWithImage = new PopupWithImage("#image-modal");
 popupWithImage.setEventListeners();
 
+// new popup for deleting a card
+const deleteCardPopup = new PopupWithConfirmation(
+  "#delete-card-modal",
+  handleDeleteCard
+);
+// set event listeners for delete card popup
+deleteCardPopup.setEventListeners();
+
+// new popup for changing avatar
+const changeAvatarPopup = new PopupWithForm(
+  "#avatar-modal",
+  handleAvatarSubmit
+);
+changeAvatarPopup.setEventListeners();
+
+// =========  USER INFO  ========= //
+
 // Instance of User Info
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const userInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
 
-// Instance of Section
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: createCard,
+// =========  APIs  ========= //
+
+// Instance of Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d918969b-47fa-4f72-8138-61be6ab55840",
+    "Content-Type": "application/json",
   },
-  "#cards__list"
-);
+});
 
-section.renderItems();
-/////////////////////////////////////////////////////////////////////
+// fetch and render initial cards
+api
+  .getInitialCards()
+  .then((result) => {
+    // Instance of Section
+    let section = new Section(
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.

Reply:
thank you